### PR TITLE
Allow for headers to be included when the gateway is requesting downstream swagger endpoint

### PR DESCRIPTION
**Pull request by duncan-G**

### CODE REVIEW

File: src/MMLib.SwaggerForOcelot/Configuration/SwaggerForOcelotUIOptions.cs [hunk 1]
```diff
@@ -32,5 +33,10 @@ public string PathToSwaggerGenerator
         /// <see cref="EndPointBasePath"/> + <see cref="SwaggerEndPointConfig.Version"/> + <see cref="SwaggerEndPointOptions.Key"/>
         /// </summary>
         public string DownstreamSwaggerEndPointBasePath { get; set; } = "/swagger/docs";
+        
+        /// <summary>
+        /// Headers to include when requesting a downstream swagger endpoint
```
Comment (Burgyn) [inline src/MMLib.SwaggerForOcelot/Configuration/SwaggerForOcelotUIOptions.cs]:
```suggestion
        /// Headers to include when requesting a downstream swagger endpoint.
```

File: src/MMLib.SwaggerForOcelot/Middleware/SwaggerForOcelotMiddleware.cs [hunk 2]
```diff
@@ -59,6 +61,14 @@ public async Task Invoke(HttpContext context)
             var endPoint = GetEndPoint(context.Request.Path);
             var httpClient = _httpClientFactory.CreateClient();
 
+            if (_options.DownstreamSwaggerHeaders != null)
```
Comment (Burgyn) [inline src/MMLib.SwaggerForOcelot/Middleware/SwaggerForOcelotMiddleware.cs]: Please can you extract it into a separate method?

Comment (duncan-G) [reply src/MMLib.SwaggerForOcelot/Middleware/SwaggerForOcelotMiddleware.cs]: Sure.

File: src/MMLib.SwaggerForOcelot/Configuration/SwaggerForOcelotUIOptions.cs (hunk 1 above)
Comment (duncan-G) [reply src/MMLib.SwaggerForOcelot/Configuration/SwaggerForOcelotUIOptions.cs]: Hi,
I've updated the ReadME fixed the above error.

**Comment by Burgyn** [comment]
Hi, thanks for your contribution.

This changes will be published to nuget in a few minutes as `v1.5.0`.

[https://www.nuget.org/packages/MMLib.SwaggerForOcelot/1.5.0](https://www.nuget.org/packages/MMLib.SwaggerForOcelot/1.5.0)

**Pull request merged**